cameo.py

```diff
@@ -78,12 +82,12 @@ def open_video_out(camera_out, width, height):
 def main(camera_in=0, camera_out=1, do_flip=False, thumbnail=False):
     current_filter = None
     keys = {
-        " ": (FilterColor,    [ (255, 0, 211) ]),
-        ")": (FilterAddImage, [ "img/smile.png" ]),
-        "a": (FilterAddImage, [ "img/applause.png" ]),
-        "t": (FilterAddText,  [ "(be right back)" ]),
-        "v": (FilterVideo,    [ "img/rick-astley-never-gonna-give-you-up-video.mp4" ]),
```

Comment:
"v" didn't seem to be used for anything. although, replacing it might not have been the best idea. Couldn't think of any keys other than b that make sense, so let me know if you can think of anything better.